libs/wallet/src/connect-dialog/connect-dialog.tsx

```diff
@@ -203,18 +197,11 @@ const ConnectorList = ({
         <ConnectDialogTitle>{t('Connect')}</ConnectDialogTitle>
         <CustomUrlInput walletUrl={walletUrl} setWalletUrl={setWalletUrl} />
         <ul data-testid="connectors-list" className="mb-6">
-          <li className="mb-4">
```

Comment:
chore (blocking): Ux team would prefer explicit separate buttons for desktop/cli, even though under the hood they will do the same thing.

```
<ConnectionOption
              type="gui"
              text={t('Desktop wallet app')}
              onClick={() => onSelect('local')}
            />
```

Reply:
As discussed lets keep the one button but update the text to 'Connect Vega wallet'